### ISSUE TITLE
Usage with v-model

### ISSUE THREAD

**Issue by Grawl** [nanostores/vue#3]
# Problem

How to use Vue NanoStores with `v-model`?

If I provide `map` store key as `v-model` value

```ts
import { map } from 'nanostores'
export const data = map({
	text: '',
})
```

```vue
<script lang='ts'>
import { defineComponent } from 'vue'
import { useStore } from '@nanostores/vue'
import { data } from './store'
export default defineComponent({
	setup() {
		const store = useStore(data)
		return { store }
	},
})
</script>
<template lang='pug'>
my-input(v-model='store.text')
</template>
```

I will get runtime warning:

```
Set operation on key "text" failed: target is readonly. 
```

Because in `map` store each key is readonly (btw its awesome), and I have to use `setKey` function to change something.

# Workaround

For now, I do this:

```vue
<script lang='ts'>
import { defineComponent } from 'vue'
import { useStore } from '@nanostores/vue'
import { data } from './store'
export default defineComponent({
	setup() {
		const store = useStore(data)
		return { data, store }
	},
})
</script>
<template lang='pug'>
my-input(
	:model-value='store.text'
	@update:model-value=`data.setKey('form', $event)`
)
</template>
```

# Proposal

Maybe there is something like [`useVModel`](https://effector.dev/ru/docs/api/effector-vue/useVModel) from `effector-vue`?

I think this will be a good API:

```vue
<script lang='ts'>
import { defineComponent } from 'vue'
import { useStore, useVModel } from '@nanostores/vue'
import { data } from './store'
export default defineComponent({
	setup() {
		const store = useStore(data)
		const textModel = useVModel(data, 'text') // store, key
		return { store, textModel }
	},
})
</script>
<template lang='pug'>
my-input(v-model='textModel')
</template>
```

**Comment by euaaaio** [comment]
Perfect issue design 🖤

Effector’s `useVModel` looks like pretty syntax sugar, but it’s not. It deeply clones object under the hood on every change. I think, it’s too expensive for just replacing two lines of code. 

Maybe with your idea of the `key` property we can make a simple wrap around writable computed 🤔

**Comment by euaaaio** [comment]
Just committed in `next`. Can you test it?

**Comment by Grawl** [comment]
`useVModel` looks exactly like I'm talking about

Cannot test it for now because I faced some Quasar+ESM issues

**Comment by Grawl** [comment]
Just tested this. It's not working :(

I tried just like my proposal. Types are ok and no errors/warnings from TS, but store is not updated.

Nor `atom` nor `map` store is not working.

I faced the same issue with Effector's `useVModel`.

**Comment by euaaaio** [comment]
Released in 0.4.1
